ramls/voucher.raml

```diff
@@ -0,0 +1,41 @@
+#%RAML 1.0
+
+title: Voucher
+version: v0.1
```

Comment:
Probably better to keep in sync with interface version i.e. `1.0`

Reply:
changed this raml as well as the invoice raml to be in sync with interface version